examples/worlds/auto_inertia_rolling_shapes.sdf

```diff
@@ -0,0 +1,187 @@
+        <constant>0.9</constant>
+        <linear>0.01</linear>
+        <quadratic>0.001</quadratic>
+      </attenuation>
+      <direction>-0.5 0.1 -0.9</direction>
+    </light>
+
+    <model name="inclined_plane">
+      <static>true</static>
+      <pose>0 0 3 0 -0.52 0</pose>
+      <link name="link">
+        <collision name="collision">
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+              <size>100 100</size>
+            </plane>
+          </geometry>
+        </collision>
+        <visual name="visual">
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+              <size>100 100</size>
+            </plane>
+          </geometry>
+          <material>
+            <ambient>0.8 0.8 0.8 1</ambient>
+            <diffuse>0.8 0.8 0.8 1</diffuse>
+            <specular>0.8 0.8 0.8 1</specular>
+          </material>
+        </visual>
+      </link>
+    </model>
+
+    <model name="ground_plane">
+      <static>true</static>
+      <pose>0 0 0 0 0 0</pose>
+      <link name="link">
+        <collision name="collision">
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+              <size>100 100</size>
+            </plane>
+          </geometry>
+        </collision>
+        <visual name="visual">
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+              <size>100 100</size>
+            </plane>
+          </geometry>
+          <material>
+            <ambient>0.8 0.8 0.8 1</ambient>
+            <diffuse>0.8 0.8 0.8 1</diffuse>
+            <specular>0.8 0.8 0.8 1</specular>
+          </material>
+        </visual>
+      </link>
+    </model>
+
+    <model name="cylinder">
+      <pose>0 0 3.6 1.57079632679 0 0</pose>
+      <link name="cylinder_link">
+        <inertial auto="true" />
+        <collision name="cylinder_collision">
+          <density>7810.0</density>
+          <geometry>
+            <cylinder>
+              <radius>0.5</radius>
+              <length>1.0</length>
+            </cylinder>
+          </geometry>
+        </collision>
+
+        <visual name="cylinder_visual">
+          <geometry>
+            <cylinder>
+              <radius>0.5</radius>
+              <length>1.0</length>
+            </cylinder>
+          </geometry>
+          <material>
+            <ambient>1 0 0 1</ambient>
+            <diffuse>1 0 0 1</diffuse>
+            <specular>1 0 0 1</specular>
+          </material>
+        </visual>
+      </link>
+    </model>
+
+    <include>
+      <name>hollow_cylinder</name>
+      <pose>0 -1.5 3.6 1.57079632679 0 0</pose>
+      <uri>https://fuel.gazebosim.org/1.0/jasmeetsingh/models/Hollow Cylinder</uri>
+    </include>
+
+    <model name="sphere">
+      <pose>0 1.5 3.6 0 0 0</pose>
+      <link name="sphere_link">
+        <inertial auto="true" />
+        <collision name="sphere_collision">
+          <densiyt>1000</density>
```

Comment:
```suggestion
          <density>1000</density>
```

Reply:
Corrected the tag in [07bd6cd](https://github.com/gazebosim/gz-sim/pull/2119/commits/07bd6cd9c29eb01cfa20804e62b289f0e66ac6f9)